Test.groovy

```diff
@@ -22,19 +22,37 @@ pipeline {
         sh "chmod u+x ./database-service/scripts/waitForService.sh && ./database-service/scripts/waitForService.sh localhost 9000"
       }
     }
-    stage('Test GraphQL engine') {
+    stage('Perform GraphQL engine tests') {
       steps {
         sh "docker-compose -f docker-compose-tests.yaml up hasura-service-tests"
       }
     }
+    // stage('Perform acceptance tests') {
+    //   steps {
+    //     sh "docker-compose -f docker-compose-tests.yaml up feature-tests"
+    //   }
+    // }
+    stage('Building specification') {
+      environment {
+        SOFTOZOR_CREDENTIALS = credentials('softozor-credentials')
+      }
+      steps {
+        script {
+          if(GIT_BRANCH == 'origin/dev' || GIT_BRANCH == 'origin/master') {
+            build job: 'backend-spec', parameters: [
+              string(name: 'BRANCH', value: GIT_BRANCH.split('/')[1])
+            ]
+          }
+        }
+      }
+    }
```

Comment:
That code there triggers the build of the specification. It calls [pickles](http://www.picklesdoc.com/) to generate a user-friendly web page which is then copied over to [softozor](http://www.softozor.ch/shopozor/backend/Index.html).